### ISSUE TITLE
Elements must meet minimum color contrast ratio thresholds

### ISSUE THREAD

**Issue by JayneSoraya** [JayneSoraya/meu-portifolio#12]
Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds

https://dequeuniversity.com/rules/axe/4.11/color-contrast?application=AxeChrome

Element Location:
.space-y-2.z-10.relative > p
<p style="opacity: 1;">Powered by <span class="text-purple-500 font-bold">JSSYSTEM</span> // Copyright © 2026. All Rights Reserved.</p>

To solve this problem, you need to fix the following:
Element has insufficient color contrast of 2.53 (foreground color: #525252, background color: #0a0a0a, font size: 9.0pt (12px), font weight: normal). Expected contrast ratio of 4.5:1

Related Node
<div class="relative min-h-screen bg-[#0a0a0a] text-white overflow-x-hidden">
